ubuntu-advantage

```diff
@@ -71,9 +16,16 @@ deb https://${1}@${REPO_URL}/ubuntu precise main
 EOF
 }
 
+is_package_installed() {
+    dpkg-query -s "$1" >/dev/null 2>&1
```

Comment:
Actually, in precise `-W` will print the package name (without version) if it's not installed. So back to `-s`